recipes/ndb_restore_data.rb

```diff
@@ -6,18 +6,22 @@
 backup_directory = "#{node['ndb']['local_backup_dir']}/#{File.basename(node['ndb']['restore']['tarball'], ".tar.gz")}"
 exclude_databases_option = ""
 exclude_tables_option = ""
+exclude_disk_objects_option = ""
 unless node['ndb']['restore']['exclude_tables_data'].empty?
     exclude_tables_option = "-e #{node['ndb']['restore']['exclude_tables_data']}"
 end
 unless node['ndb']['restore']['exclude_databases_data'].empty?
     exclude_databases_option = "-x #{node['ndb']['restore']['exclude_databases_data']}"
 end
+unless node['ndb']['restore']['exclude_disk_objects'].eql?("false")
+    exclude_disk_objects_option = "-d"
```

Comment:
It's actually only the restore metadata which creates the disk objects. Restore data and restore sql do not have this option.

Reply:
Reverted the changes - though they didn't have any effect since the restore script ignore the parameter for the data and sql options.